index.html

```diff
@@ -115,6 +115,7 @@ <h3><a id="tools" href="#tools">Tools and plugins</a></h3>
             <li>Drupal users, <a href="https://www.drupal.org/project/naked">get the module</a>!</li>
             <li>If you are game you can use <a href="https://weblog.200ok.com.au/">Ben Buchanan</a>’s <a href="https://bitbucket.org/snippets/200ok/dKpk">JavaScript snippet</a>. Works anywhere, removes everything, buyer beware!</li>
             <li>If you are a Rails developer, check out the <a href="https://github.com/danielsellergren/css_naked_day_rails">css_naked_day_rails</a> gem by <a href="https://danielsellergren.com/">Daniel Sellergren</a>.</li>
+            <li>Use <a href="https://gist.github.com/alifeee/75df204c2be3815806e1d4be47e54d08">this bash script</a> to remove stylesheets, style tags, and inline styles from multiple raw HTML or other files.</li>
```

Comment:
Please use a similar wording as the other points, which start with a person or a condition.

Also, “`style` elements” would be most precise.

For “stylesheets,” I suggest the spelling “style sheets,” but I don’t think we established a convention on the site. (@arkhi, any views?)

Reply:
I have changed as per your suggestions 👍